app/locales/en/translations.js

```diff
@@ -397,6 +400,7 @@ export default {
                 pending_post: 'is publicly available and searchable but is subject to removal by a moderator',
                 accepted: 'has been accepted by a moderator and is publicly available and searchable',
                 rejected: 'has been rejected by a moderator and is not publicly available or searchable',
+                withdrawn: 'This preprint has been withdrawn by the author(s).',
```

Comment:
Same as above, we may want to use generic 'paper' language instead of preprint, but needs product confirmation.